pkg/fs/apkfs.go

```diff
@@ -235,8 +235,18 @@ func (a *APKFS) Stat(path string) (fs.FileInfo, error) {
 	}
 	return &apkFSFileInfo{file: file, name: file.name[strings.LastIndex(file.name, "/"):]}, nil
 }
-
+func correctApkFSPath(path string) string {
+	if path == "." {
```

Comment:
```suggestion
	path = filepath.Clean(path)

	if path == "." {
```

Move the cleaning before checking.